_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -964,19 +1010,20 @@ Next, create a `terragrunt.hcl` file in `infrastructure-live`. It should go unde
 
 ----
 infrastructure-live
-  └ root
+  └ prod
```

Comment:
Not sure why the change? this needs to setup the root account before, `prod` references the production environment where the customer run their web apps.

Reply:
oh, interesting. I looked at the `for-production` example code that I believe must have been generated by the RefArch 2.0 and didn't see `root` folder there - I could find all the relevant files in there. How come this one was called `root`?

Reply:
Because we don't generate a `root` baseline in the Ref Arch

Reply:
Hm, ok. I'm happy to change it back - it was a little bit confusing for me, but I think your logic makes sense!